components/Accordion/src/index.scss

```diff
@@ -52,6 +52,8 @@
 .denhaag-accordion__title {
   --utrecht-heading-5-margin-block-start: 0;
   --utrecht-heading-5-margin-block-end: 0;
+  --utrecht-heading-5-font-family: var(--denhaag-accordion-font-family);
+  --utrecht-heading-5-font-weight: 400;
```

Comment:
This styling of `heading-5` tokens seems like a very roundabout way to style the h3 of the accordion differently.

Seems to me there a much simpler and more flexible approach is possible: style the `<button>` to not have its own style, but to inherit font-family, color, etc. For example:

```css
.denhaag-accordion__title-button {
  color: inherit;
  font-family: inherit;
  font-size: inherit;
  line-height: inherit;
  width: 100%;
  /* etc */
}
```